bigbluebutton-html5/imports/ui/components/lock-viewers/styles.scss

```diff
@@ -0,0 +1,118 @@
+@import '/imports/ui/stylesheets/mixins/focus';
+@import '/imports/ui/stylesheets/variables/_all';
+@import "/imports/ui/components/modal/simple/styles";
+
+:root{
+  --modal-margin: 3rem;
+}
+
+.title {
+  color: var(--color-gray-dark);
+  font-weight: 400;
+  font-size: var(--lg-padding-x);
+  margin: 0;
+  margin-bottom: var(--jumbo-padding-y);
+  text-align: center !important;
+}
+
+.form {
+  display: flex;
+  flex-flow: column;
+}
+
+.container{
+    margin: var(--modal-margin);
+    margin-top: var(--indicator-padding);
+    margin-bottom: var(--lg-padding-x);
+}
+
+.subHeader{
+  color: var(--color-gray-label);
+  font-size: var(--font-size-base);
+  margin-bottom: var(--jumbo-padding-y) !important;
+}
+
+.modal {
+    @extend .modal;
+    padding: var(--jumbo-padding-y);
+ }
+
+.overlay {
+    @extend .overlay;
+  }
+
+.description{
+    text-align: center;
+    color: var(--color-gray);
+}
+.row {
+  display: flex;
+  flex-flow: row;
+  flex-grow: 1;
+  justify-content: space-between;
+  margin-bottom: var(--md-padding-x);
+}
+
+.col {
+  display: flex;
+  flex-grow: 1;
+  flex-basis: 0;
+  margin-right: var(--md-padding-x);
+}
+
+.label {
+  color: var(--color-gray-label);
+  font-size: var(--font-size-small);
+  margin-bottom: var(--lg-padding-y);
+}
+
+.formElement {
+  position: relative;
+  display: flex;
+  flex-flow: column;
+  flex-grow: 1;
+}
+
+.pullContentRight {
+  display: flex;
+  justify-content: flex-end;
+  flex-flow: row;
+}
+
+.title {
+  font-weight: bold;

```

Comment:
Is the `font-weight`  for `.title` suppose to be `bold` or `400`?  It has a different value on line 11. Also, The `text-align` is duplicated below.